src/renderer/app/services/data.service.ts

```diff
@@ -85,6 +86,10 @@ export class DataService extends Logger {
     };
   }
 
+  public formatQueryParams(requestParams: Transaction['request']['queryParams']) : { name: string; value: string }[] {
+    return this.formatQueryParamsWithPrefix('', requestParams);
+  }
+
```

Comment:
Do we really need this intermediate function? I guess you could use directly `this.formatQueryParamsWithPrefix('', requestParams)` on line 75
Also, if kept, I think this method should be private.

Reply:
I have introduced this mostly for readability reasons. I can make it go, if you like

Reply:
Let's keep it then, but can you make it `private`?

Reply:
Done